.changeset/spicy-beds-give.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/icon': minor
+---
+
+Adds Circle to glyph set.
```

Comment:
Did you also update coin?

Reply:
I did not, not sure why the checksum changed